gammapy/data/observations.py

```diff
@@ -554,7 +557,6 @@ def write(
             Path for the output file.
         overwrite : bool, optional
             Overwrite existing file. Default is False.
-        format : {"gadf"}
             Output format, currently only "gadf" is supported. Default is "gadf".
```

Comment:
Maybe one can put this line at the start of the docstring, instead than in the middle...

Reply:
Oh I actually did not want to remove it😅
Was a mistake